examples/web/rust/index.html

```diff
@@ -423,11 +455,8 @@ <h2 class="title is-4 has-text-centered">Get</h2>
                     </div>
                 </div>
             </form>
-            <div class="mt-6">
```

Comment:
we need to see the `get` response here

Reply:
It shows up in the console

Reply:
Does that work?

Reply:
I'm afraid it's a bit too verbose and it may get lost.

Some of the options that I can come up with:
- add log level (so low-level connection stuff would be in `DEBUG`, hidden by default), seems like a lot of work
- keep low-level wire protocol binary logs as `console.log`, only print user-facing stuff (errors and received value)
- have a separate "log" for debugging, an element, where `get` value is printed and a notification sliding out when error is encountered

Reply:
FWIW, here's what I see:
```
2024-11-06T16:39:27.285Z  [info] got value
2024-11-06T16:39:27.285Z  [info] received `get` response: 0,11,0,1,8,98,97,114,115,102,115,100,102
2024-11-06T16:39:27.284Z  [info] reading `get` response...
2024-11-06T16:39:27.283Z  [info] creating `get` stream...
```

So not actual value gotten

Reply:
yeah, ok, that makes sense.
Does this work to highlight output?
<img width="660" alt="Screenshot 2024-11-06 at 11 39 17 AM" src="https://github.com/user-attachments/assets/9e3dbb99-4609-4885-9249-911969729e7d">

I did just fix the issue with the value not displaying

Reply:
I could also add a "level" dropdown and have it set to "info" by default